mappings/InventoryID-Bibtex.tsv

```diff
@@ -3432,401 +3432,395 @@ InventoryID	BibtexKey	Source
+2925	alpher_yir-yoront_1991	er
+2926	sutton_diversity_1976	er
+2927	sutton_diversity_1976	er
+2928	dixon_mbabaram_1991	er
+2929	verstraete_umbuygamu_2017-1	er
+2930	sommer_kunjen_1969	er
+2931	schebeck_texts_1974	er
+2932	amery_kulurdu_2013	er
+2933	hercus_nukunu_1992	er
+2934	hercus_grammar_1999	er
+2935	ogrady_two_2001	er
+2936	eira_fragments_2010	er
+2937	platt_outline_1972	er
+2938	holmer_linguistic_1983	er
+2939	holmer_linguistic_1983	er
+2940	bell_sketch_2003	er
+2941	holmer_linguistic_1983	er
+2942	holmer_linguistic_1983	er
+2943	kite_duungidjawu_2004	er
+2944	holmer_linguistic_1983	er
+2945	holmer_linguistic_1983	er
+2946	breen_bularnu_1988	er
+2947	breen_wakaya_1974	er
+2948	breen_warluwarra_2015	er
+2949	breen_wakaya_1974	er
+2950	kirton_seven_1978	er
+2951	simpson_learners_2002	er
+2952	marsh_grammar_1976	er
+2953	burgman_manyjilyjarra-english_2009	er
+2954	wangka_maya_pilbara_aboriginal_language_centre_putijarra-english_2004	er
+2955	jones_grammar_2011	er
+2956	burridge_yulparija_1996	er
+2957	ogrady_languages_1966	er
+2958	ogrady_languages_1966	er
+2959	hansen_core_1978	er
+2960	ogrady_languages_1966	er
+2961	ogrady_languages_1966	er
+2962	glass_pitjantjatjara_1970	er
+2963	huttar_notes_1976	er
+2964	glass_pitjantjatjara_1970	er
+2965	goddard_grammar_1985	er
+2966	hunter_language_2011	er
+2967	hunter_language_2011	er
+2968	hercus_maljangapa-wadigali_1989	er
+2969	hercus_yarli_2004	er
+2970	hercus_yarli_2004	er
+2971	wood_yuulngu_1978	er
+2972	wilkinson_djambarrpuyngu_1991	er
+2973	wood_yuulngu_1978	er
+2974	wilkinson_djambarrpuyngu_1991	er
+2975	wilkinson_djambarrpuyngu_1991	er
+2976	waters_djinang_1980	er
+2977	waters_djinang_1989	er
+2978	wilkinson_djambarrpuyngu_1991	er
+2979	heath_basic_1980	er
+2980	bowe_yorta_1999	er
+2981	hercus_victorian_1986	er
+2982	wafer_handbook_2008	er
+2983	wafer_handbook_2008	er
+2984	jones_darkinyung_2008	er
+2985	troy_sydney_1992	er
+2986	troy_sydney_1992	er
+2987	lissarrague_salvage_2006	er
+2988	blake_dialects_2011	er
+2989	lissarrague_grammar_2010	er
+2990	crowley_phonological_1976	er
+2991	lissarrague_dhanggati_2007	er
+2992	morphy_djapu_1983	er
+2993	besold_language_2013	er
+2994	besold_language_2013	er
+2995	besold_language_2013	er
+2996	besold_language_2013	er
+2997	besold_language_2013	er
+2998	hercus_victorian_1986	er
+2999	besold_language_2013	er
+3000	besold_language_2013	er
+3001	round_description_2014	er
+3002	round_kayardild_2013	er
+3003	round_word_2011	er
+3004	cook_sketch_2017	er
+3005	osborne_tiwi_1974	er
+3006	cook_wagiman_1987	er
+3007	rumsey_intra-sentence_1982	er
+3008	capell_comparative_1984	er
+3009	mcgregor_gunin/kwini_1993	er
+3010	carr_wunambal:_2000	er
+3011	capell_comparative_1984	er
+3012	capell_comparative_1984	er
+3013	clendon_worrorra:_2014	er
+3014	capell_comparative_1984	er
+3015	merlan_grammar_1994	er
+3016	merlan_grammar_1994	er
+3017	austin_guwamu_1980	er
+3018	nash_preliminary_1979	er
+3019	blake_dialects_2011	er
+3020	verstraete_mbarrumbathama_2018	er
```

Comment:
```suggestion
3020	verstraete_mbarrumbathama_2018	er

```

Reply:
(adds missing newline to end of file)

Reply:
@drammock isn't this optional in RFC 4180 2.2? :)

Honestly, I will be happy to put line endings at the end of all CSV files if we actually move over from TSV to CSV -- it's so much easier to deal with as the default in most programming languages and working formats.

https://tools.ietf.org/html/rfc4180

2.  Definition of the CSV Format
   2.  The last record in the file may or may not have an ending line
       break.  For example:

       aaa,bbb,ccc CRLF
       zzz,yyy,xxx

Reply:
I don't object to switching these to CSVs (in a subsequent PR).  I agree that it's optional according to the standard, but git and GitHub complain if text files don't end with newlines, and it's slightly safer for anyone who wants to work with the files using typical bash tools, so I think we should standardize on including newline at EOF.